packages/mobile/locales/es-419/nuxVerification2.json

```diff
@@ -1,4 +1,5 @@
 {
+  "getVerified": "~~Verify Phone Number",
```

Comment:
Just as a question, what `~~` is for?

Reply:
That's just a signal for me to know these are placeholders until the actual translations come in from our contractor

Reply:
That's just a signal for me to know these are placeholders until the actual translations come in from our contractor